universe/common/helper/spaces.go

```diff
@@ -27,6 +27,7 @@ type SpaceTemplate struct {
 	Label           *string              `json:"label"`
 	SpaceAttributes []*entry.Attribute   `json:"space_attributes"`
 	Spaces          []*SpaceTemplate     `json:"spaces"`
+	RandomSpaces    []*SpaceTemplate     `json:"random_spaces"`
```

Comment:
are you sure that it supposed to be part of `SpaceTemplate` not `WorldTemplate`?

Reply:
yes, they are supposed to be children of the world